src/mysql.c

```diff
@@ -905,6 +905,8 @@ static int mysql_read(user_data_t *ud) {
 
     } else if (strncmp(key, "Slow_queries", strlen("Slow_queries")) == 0) {
       derive_submit("mysql_slow_queries", NULL, val, db);
+    } else if (strncmp(key, "Uptime", strlen("Uptime")) == 0) {
```

Comment:
Hi,

This line is wrong. Mysql output has `Uptime` and `Uptime_since_flush_status` variables, and this check matches them both.